cypress/integration/2-advanced-examples/mainPage_spec.js

```diff
@@ -0,0 +1,64 @@
+describe('Under-a-tack mainPage', () => {
+
+    beforeEach(() => {
+        cy.intercept("GET", "http://localhost:3001/api/v1/images/:16" , {
```

Comment:
If we're on the main page, we probably just need to intercept the homepage ("/") path.
We could probably just stub back the `url` property of each image for the time being if we're just testing the display of the images on the page.

Reply:
Thank you,  addressed

Reply:
Awesome, thank you.